cli/flags.rs

```diff
@@ -731,6 +739,20 @@ Future runs of this module will trigger no downloads or compilation unless
     )
 }
 
+fn upgrade_subcommand<'a, 'b>() -> App<'a, 'b> {
+  SubCommand::with_name("upgrade")
+    .about("Upgrade deno executable to newest version")
+    .long_about(
+      "Upgrade deno executable to newest available version.
+      
+  Checks for newest available version of deno and if higher than
+  local version, then downloads and installs newest version in place
+  of the current one.
```

Comment:
Don't indent here. Mention that the executable is downloaded from  https://github.com/denoland/deno/releases